File: tests/openmp_examples/program_control/cancellation.2.c
#pragma omp task shared(found) if(level < 10)
#pragma omp atomic write
#pragma omp cancel taskgroup
#pragma omp task shared(found) if(level < 10)
#pragma omp atomic write
#pragma omp cancel taskgroup
#pragma omp taskwait
#pragma omp parallel shared(found, tree, value)
#pragma omp masked
#pragma omp taskgroup

File: tests/openmp_examples/program_control/cancellation.2.f90
!$omp task shared(found) if(level<10)
!$omp atomic write
!$omp end atomic
!$omp cancel taskgroup
!$omp end task
!$omp task shared(found) if(level<10)
!$omp atomic write
!$omp end atomic
!$omp cancel taskgroup
!$omp end task
!$omp taskwait
!$omp parallel shared(found, tree, value)
!$omp masked
!$omp taskgroup
!$omp end taskgroup
!$omp end masked
!$omp end parallel
